code/modules/mob/living/basic/pets/parrot/parrot_ai/ghost_parrot_controller.dm

```diff
@@ -0,0 +1,39 @@
+/// Used for ghost poly.
+/datum/ai_controller/basic_controller/parrot/ghost
+	planning_subtrees = list(
+		/datum/ai_planning_subtree/parrot_as_in_repeat,
+		/datum/ai_planning_subtree/possess_humans,
+		/datum/ai_planning_subtree/hoard_items,
+	)
+
+///subtree to possess humans
+/datum/ai_planning_subtree/possess_humans
+	///chance we go possess humans
+	var/possess_chance = 80
```

Comment:
```suggestion
	var/possess_chance = 2 
```
Last thing. I had it at a high value just for testing I forgot to set it to a fairer number